docs/src/guide/load-balancer-annotations.md

```diff
@@ -0,0 +1,147 @@
+# Load Balancer Annotations
+
+This file defines a list of [Service][4] `type: LoadBalancer` annotations which are
+supported by the `oci-cloud-controller-manager`.
+
+All annotations are prefixed with `service.beta.kubernetes.io/` or `oci.oraclecloud.com/` or `oci-network-load-balancer.oraclecloud.com/` (for OCI Network Load Balancer specific annotations). For example:
+
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: nginx-service
+  annotations:
+    oci.oraclecloud.com/load-balancer-type: "lb"
+    service.beta.kubernetes.io/oci-load-balancer-shape: "400Mbps"
+    service.beta.kubernetes.io/oci-load-balancer-subnet1: "ocid..."
+    service.beta.kubernetes.io/oci-load-balancer-subnet2: "ocid..."
+    oci.oraclecloud.com/loadbalancer-policy: "IP_HASH"
+    oci.oraclecloud.com/oci-network-security-groups: "ocid1..."
+spec:
+  ...
+```
+
+## Load balancer Type Annotation
```

Comment:
Re-orgnanize the annotations by api group and pay particular attention to sec list vs nsg.